File: intellij-community/platform/lang-impl/src/com/intellij/bootRuntime/command/Processor.java
// Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
package com.intellij.bootRuntime.command;

import org.jetbrains.annotations.NotNull;

public class Processor {

  public static void process(@NotNull RuntimeCommand... commands) {
    for (RuntimeCommand c : commands) {
      c.actionPerformed(null);
    }
  }
}

File: plugins/lombok/lombok.k1/src/org/jetbrains/kotlin/lombok/processor/Processor.kt
/*
 * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
 * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
 */

package org.jetbrains.kotlin.lombok.processor

import org.jetbrains.kotlin.descriptors.ClassDescriptor
import org.jetbrains.kotlin.load.java.lazy.LazyJavaResolverContext

/**
 * Generates synthetic parts for [ClassDescriptor]
 */
interface Processor {

    context(LazyJavaResolverContext)
    fun contribute(classDescriptor: ClassDescriptor, partsBuilder: SyntheticPartsBuilder)

}
